app/schema/skip_condition.py

```diff
@@ -1,5 +1,11 @@
+import copy
 
 
 class SkipCondition(object):
     def __init__(self):
         self.when = None
+
+    def copy_as_dict(self):
```

Comment:
Does the person using this method need to know it's a copy, or can we just call it `as_dict`?